tsconfig.json

```diff
@@ -14,6 +14,11 @@
     "isolatedModules": true,
     "jsx": "preserve"
   },
-  "include": ["next-env.d.ts", "**/*.ts", "**/*.tsx"],
+  "include": [
+    "next-env.d.ts",
+    "**/*.ts",
+    "**/*.tsx",
+    "src/pages/_document.tsx"
```

Comment:
don't then we need this since above 2 include everything.

Reply:
I can remove the `"src/pages/_document.tsx"`, but the other ones get added back on `next build` anyway, or I could leave an empty array and disable the lint rule that checks it.